source/includes/gcp/_images.md

```diff
@@ -112,7 +112,7 @@ Attributes | &nbsp;
 
 ```shell
 curl -H "MC-Api-Key: your_api_key" \
-    "https://api.your.cloudmc/v1/services/gcp/test-area/images/4658005417542122143"
+    "https://api.your.cloudmc/api/v1/services/gcp/test-area/images/4658005417542122143"
```

Comment:
Should `api.your.cloudmc` be `cloudmc_endpoint` instead?

Reply:
Fixed in commit b72186371f3f9263f21b901b1608cb3131183861